grc/utils/application_files.py

```diff
@@ -14,7 +15,7 @@ def create_or_download_attachments(self, reference_number, application, download
         try:
             file_name = reference_number + '.zip'
 
-            data = AwsS3Client().download_object(file_name)
+            data = None if os.getenv('FLASK_ENV', '') == 'development' else AwsS3Client().download_object(file_name)
```

Comment:
Just wondering what this is for?
Is there a reason we don't want this to run on our laptops?

Reply:
AWS is effectively being used as a file cache, which is a pain when testing locally as a cached file won't reflect changes in the code, this bypasses it

Reply:
Ah yes, I get it 🙂 
That makes sense